.tests/integration/reference/SSU72.gff

```diff
@@ -0,0 +1,12 @@
+##gff-version 3
```

Comment:
Do we need this file?

Reply:
It is not needed to run but if we need to redo the index for a new version it is good to have

Reply:
I understand your reasoning but it should not be part of the repo. Otherwise, we could argue to keep other files around which we used to generate test data and that would result in a huge amount of unnecessary data.